module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaPasswordSingleSignOnCredential.md

```diff
@@ -9,41 +19,60 @@ schema: 2.0.0
 # New-EntraBetaPasswordSingleSignOnCredential
 
 ## Synopsis
-Creates the password SSO credentials
+
+Creates the password Single-Sign-On (SSO) credentials.
 
 ## Syntax
 
-```
-New-EntraBetaPasswordSingleSignOnCredential -ObjectId <String>
- -PasswordSSOCredential <PasswordSSOCredentials> [<CommonParameters>]
+```powershell
+New-EntraBetaPasswordSingleSignOnCredential 
+ -ObjectId <String>
+ -PasswordSSOCredential <PasswordSSOCredentials> 
+ [<CommonParameters>]
 ```
 
 ## Description
-This cmdlet enables users to create their Password Single-sign-on credentials for an application which they are part of.
+
+This cmdlet enables users to create their Password Single-Sign-On credentials for an application that they're part of. Specify `ObjectId` and `PasswordSSOCredential` parameters to create an SSO credentials.
 Admin could create the group credentials as well.
 
 ## Examples
 
-### New password single-sign-on credentials
+### Example 1: New password single-sign-on credentials
+
+```powershell
+Connect-Entra -Scopes 'Application.ReadWrite.All and Directory.Read.All, Directory.ReadWrite.All'
+$credentials = New-Object -TypeName Microsoft.Open.MSGraph.Model.PasswordSSOCredentials
```

Comment:
Confirm loaded types - using the latest installation of Beta module:

![image](https://github.com/user-attachments/assets/e5d159c2-fee6-48b9-bde8-e5ade4352ca0)

Reply:
Its working on my end.

![Screenshot Success](https://github.com/user-attachments/assets/9796e6fc-3a52-4310-8e2d-9bd1c3c7f5c4)